### PR TITLE
Display Total Stats on Mentor Dashboard

### PR DESCRIPTION
**Pull request by pranjal3060**
Added Total Commits and Total Pull Requests : Calculated and displayed the total number of commits and pull requests across all projects.

Added Total Projects and Approved Projects : Included the count of total projects and approved projects in the overall statistics section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -125,3 +125,23 @@ The development for the next KWoC website continues in the default branch.
 ---
 
 > Please update this documentation whenever changes are made to this project or any other relevant project that may affect this one. Future humans will praise you.
+
+## Changes Overview
```
Comment (harshkhandeparkar) [inline README.md]: You don't need to add anything here :)

File: src/data/FAQs.ts [hunk 2]
```diff
@@ -143,7 +143,7 @@ export const FAQs = [
   {
     question: "How are commits counted?",
     answer: [
-      "Commits are counted only after your PR is merged in the mentor's repository. The PR must be opened using the Github account you logged in with and your username should not be changed during the KWoC coding period.",
```
Comment (harshkhandeparkar) [inline src/data/FAQs.ts]: Is this a mistake?

**Review by harshkhandeparkar — CHANGES_REQUESTED** [review]

**Comment by harshkhandeparkar** [comment]
@pranjal3060 Can you send a screenshot of the full mentor dashboard with the changes? Thanks.

**Comment by harshkhandeparkar** [comment]
@pranjal3060 also run `pnpm build` locally and fix any errors that are shown.

**Comment by pranjal3060** [comment]
> @pranjal3060 Can you send a screenshot of the full mentor dashboard with the changes? Thanks.

image

**Comment by pranjal3060** [comment]
> @pranjal3060 Can you send a screenshot of the full mentor dashboard with the changes? Thanks.

image

File: src/pages/MentorDashboard.tsx [hunk 3]
```diff
@@ -45,9 +47,26 @@ function MentorDashboard() {
...
+  let totalPRs = 0;
+  let totalCommits = 0;
+  let totalProjects = 0;
+  let approvedProjects = 0;
+
+  if (dashboard !== null) {
+    dashboard.projects.forEach((project) => {
+      totalPRs += project.pulls.length;
+
+      project.commits.forEach((commit) => {
```
Comment (harshkhandeparkar) [inline src/pages/MentorDashboard.tsx]: Shouldn't this be `project.pulls`?

File: src/pages/MentorDashboard.tsx [hunk 4]
```diff
@@ -84,16 +103,56 @@ function MentorDashboard() {
...
+                  <span>Total Projects:</span>
+                </div>
+                <p className="font-bold text-base">{totalProjects}</p>
+              </div>
+              <div className="flex gap-2 items-center justify-between">
+                <div className="flex gap-2 items-center text-sm font-semibold">
+                  <IoPersonSharp />
+                  <span>Approved Projects:</span>
+                </div>
+                <p className="font-bold text-base">{approvedProjects}</p>
```
Comment (harshkhandeparkar) [inline src/pages/MentorDashboard.tsx:132]: Can we display `{approvedProjects} / {totalProjects}` here?

**Review by harshkhandeparkar — CHANGES_REQUESTED** [review]
The build is still failing. Please run `pnpm build` and fix the issues.

**Comment by harshkhandeparkar** [comment]
> > @pranjal3060 Can you send a screenshot of the full mentor dashboard with the changes? Thanks.
> 
> image

Can you send one with some pull requests displayed?

File: src/pages/MentorDashboard.tsx (hunk 3 above)
Comment (harshkhandeparkar) [reply src/pages/MentorDashboard.tsx]: nvm, my bad.

File: src/pages/MentorDashboard.tsx (hunk 4 above)
Comment (harshkhandeparkar) [reply src/pages/MentorDashboard.tsx:132]: it's already displayed sorry

**Comment by harshkhandeparkar** [comment]
Thanks @pranjal3060!

**Pull request merged**